skimage/measure/entropy.py

```diff
@@ -0,0 +1,33 @@
+from scipy.stats import entropy as scipy_entropy
+from .._shared.utils import assert_nD
+
+
+def shannon_entropy(image):
```

Comment:
Another slight preference would be for the name to just be `entropy`. At which point we are just a very thin wrapper around scipy, but that's ok.